libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/NavigationMapRoute.java

```diff
@@ -589,6 +605,26 @@ public Builder withVanishingRouteLineUpdateIntervalNano(
       return this;
     }
 
+    /**
+     * Indicates the size of the bounding box used to determine which route line was clicked. Upon
+     * a map click an attempt will be made to find a route line intersecting the bounding box. This
+     * is used to select alternative route lines on the map.
+     *
+     * Passing a large integer values through this method means that a user doesn't
+     * have to be very accurate with tapping on an actual route line because the invisible
+     * box is large and thus, the chances that a portion of a route line running through the
+     * box's area are high. A smaller rectangle will decrease the chances.
+     *
+     * A device independent pixel value will be calculated based on the input padding provided.
```

Comment:
```suggestion
     * Expects DP as an input. A device specific pixel value will be calculated based on the input padding provided.
```

If I understand correctly, it's the other way around. We're expecting device-independent DP and converting it to pixels internally.

Reply:
I'm thinking whether it wouldn't be better to expose raw pixels setter instead of DP setter and document that a good practice would be for the value to be adjusted by pixel density by the developer. This is typically how the Android SDK operates.

Reply:
I think that's a better strategy although we still need a default value since most developer's probably won't use this.  I modified the code so that if no value is provided it will use default combined with the device's density to create a default value.